src/modules/06.md

```diff
@@ -1,4 +1,5 @@
 ---
+module_number: '06'
```

Comment:
I _believe_ we could also get the file title from [Eleventy supplied data](https://www.11ty.dev/docs/collections/#collection-item-data-structure). I also haven't done that in the overview page, but perhaps we could refactor this in the future. Consider this comment optional.

Reply:
I found out a way to use Eleventy supplied data in the overview, as demonstrated in the linked documentation page. Unfortunately however, this doesn't work within individual layout pages. At least, I am not able to find out how. Do you have an idea?

Reply:
Forger it! I found a way. You can access that on the `page` object. E.g. `{{ page.fileSlug }}`